### DISCUSSION TITLE
Monthly repeating SIP transactions.

### DISCUSSION THREAD

**Discussion by billybutcher2019** [ananthakumaran/paisa#234] [Q&A]
Hello,

I have gone through the documentation, but I could not find any examples related to this. I wanted to do a fixed amount for the mutual fund that automatically computes the units based on the NAV of the day and updates my ledger. Since Paisa already has a way of looking up the market price, I wanted to know how to implement this.

Thank you.

**Comment by ananthakumaran** [comment]
All the transactions have to be entered with the cost & units. Paisa does have the price info, but I also believe the ledger should be self-contained. The price info is mostly used for charting and paisa will also warn if the cost you entered is not the same as what it gets from other places.

**Reply by billybutcher2019** [reply]
I see. Thanks for the reply.
